.github/CONTRIBUTING.md

```diff
@@ -114,17 +114,21 @@ Draft pull requests are also welcome to get feedback early on, or if there is so
 
 Apart from the descriptors, it will usually involve modifying files such as [.automation/build.py](https://github.com/oxsecurity/megalinter/blob/main/.automation/build.py)
 
-In order to be able to run locally a server that serves all the documentation and make the testing as real as possible you will have to run at least 2 commands.
+In order to be able to run locally a server that serves all the documentation and make the testing as real as possible you should setup a virtual environment and.
```

Comment:
Is the sentence intended to finish in "and."?

Reply:
Oops :(